### PR TITLE
Changes in SDK Code Snippets

### PR DESCRIPTION
**Pull request by clarifai-m**

### CODE REVIEW

File: code_snippets/python-sdk/inference/diff_baseurl.py [hunk 1]
```diff
@@ -22,15 +22,13 @@
...
-
 # Example for prediction through Filepath:
 # model_prediction = Model(model_url).predict_by_filepath(image_filepath, input_type="text")
 
-
 model_url = "https://clarifai.com/cohere/embed/models/cohere-embed-english-v3_0"
 
 # You can pass the new base url as paramater while initializing the Model object
-model_prediction = Model(url=model_url, pat="YOUR_PAT",base_url="New Base URL").predict_by_bytes(
+model_prediction = Model(url=model_url, pat="your_pat",base_url="New Base URL").predict_by_bytes(
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/diff_baseurl.py]: What difference does it make? In code_snippets/python-sdk/inference/batch_predict.py it **YOUR_PAT** but here **your_pat**, why?

File: code_snippets/python-sdk/inference/text_class_llm.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+
+# Example for prediction through URL:
+# model_prediction = model.predict_by_url(url, input_type="text")
 
+# Example for prediction through Filepath
+# model_prediction = Model(model_url).predict_by_filepath(filepath, input_type="text")
 
 # Model Predict
-model_prediction = Model("https://clarifai.com/openai/chat-completion/models/GPT-4").predict_by_bytes(prompt.encode(), input_type="text")
+model_prediction = Model("model_url", pat="your_pat").predict_by_bytes(prompt.encode(), input_type="text")
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/text_class_llm.py]: **model_url** is a variable not a string. And check out regards to **pat** in previous comment

File: code_snippets/python-sdk/inference/audioToText.ts [hunk 3]
```diff
@@ -29,22 +29,21 @@ import { Model } from "clarifai-nodejs";
...
+                                    inputType: "Text"
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/audioToText.ts]: These examples are different from the code below, why?

File: code_snippets/python-sdk/inference/imageTextToText.ts [hunk 4]
```diff
@@ -37,22 +37,21 @@ const multiInputs = Input.getMultimodalInput({
...
+                                    inputType: "Text"
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/imageTextToText.ts]: These examples are different from the code below, why?

File: code_snippets/python-sdk/inference/imageToImage.ts [hunk 5]
```diff
@@ -29,22 +29,21 @@ import fs from "fs";
...
+                                    inputType: "Text"
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/imageToImage.ts]: These examples are different from the code below, why?

File: code_snippets/python-sdk/inference/imageToText.ts [hunk 6]
```diff
@@ -31,22 +31,21 @@ const modelUrl =
...
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
                                 });
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/imageToText.ts]: These examples are different from the code below, why?

File: code_snippets/python-sdk/inference/text_gen_llm.py [hunk 7]
```diff
@@ -5,6 +5,14 @@
 # You can set the model using model URL or model ID.
 model_url="https://clarifai.com/openai/chat-completion/models/GPT-4"
 
+# The predict API gives the flexibility to generate predictions for data provided through URL, Filepath and Bytes format. 
+
+# Example for prediction through URL:
+# model_prediction = model.predict_by_url(url, input_type="text")
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/text_gen_llm.py]: Instead of **model** should have used **Model(model_url)** I think

Comment (srikanthbachala20) [reply code_snippets/python-sdk/inference/text_gen_llm.py]: Check all **.py** files for this issue

File: code_snippets/python-sdk/inference/text_gen_llm_inference.py [hunk 8]
```diff
@@ -18,6 +18,14 @@
 # You can set the model using model URL or model ID.
 model_url="https://clarifai.com/meta/Llama-3/models/llama-3_1-8b-instruct"
 
+# The predict API gives the flexibility to generate predictions for data provided through URL, Filepath and Bytes format.
+
+# Example for prediction through URL:
+# model_prediction = model.predict_by_url(url, input_type="text", inference_params=inference_params)
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/text_gen_llm_inference.py]: As mentioned in previous comment

File: code_snippets/python-sdk/inference/text_gen_llm_inference.py [hunk 9]
```diff
@@ -18,6 +18,14 @@
...
+
+# Example for prediction through Filepath
+# model_prediction = Model(model_url).predict_by_filepath(filepath, input_type="image", inference_params=inference_params)
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/text_gen_llm_inference.py]: **input_type** should be **text** right?

File: code_snippets/python-sdk/inference/visualClassifier.ts [hunk 10]
```diff
@@ -37,23 +37,21 @@ const model = new Model({
...
+        Example for prediction through Bytes:
+        const modelPrediction = await model.predictByBytes({
+                                    inputBytes: Bytes,
+                                    inputType: "Text"
+                                });
+
+        Example for prediction through Filepath:
+        const modelPrediction = await model.predictByFilepath({
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/visualClassifier.ts]: **inputType** should be **image**

File: code_snippets/python-sdk/inference/visualDetectorImage.ts [hunk 11]
```diff
@@ -37,23 +37,22 @@ const detectorModel = new Model({
...
+        Example for prediction through Bytes:
+        const modelPrediction = await model.predictByBytes({
+                                    inputBytes: Bytes,
+                                    inputType: "Text"
+                                });
 
+        Example for prediction through Filepath:
+        const modelPrediction = await model.predictByFilepath({
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/visualDetectorImage.ts]: **inputType** should be **image**

File: code_snippets/python-sdk/inference/visualDetectorVideo.ts [hunk 12]
```diff
@@ -22,6 +22,22 @@ const SAMPLE_MS = 2000;
...
+        Example for prediction through Bytes:
+        const modelPrediction = await model.predictByBytes({
+                                    inputBytes: Bytes,
+                                    inputType: "Text"
+                                });
+
+        Example for prediction through Filepath:
+        const modelPrediction = await model.predictByFilepath({
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/visualDetectorVideo.ts]: **inputType** should be **video**

File: code_snippets/python-sdk/inference/visualEmbedder.ts [hunk 13]
```diff
@@ -28,23 +28,22 @@ import { Model } from "clarifai-nodejs";
...
+                                    inputType: "Text"
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/visualEmbedder.ts]: **inputType** should be **image**

File: code_snippets/python-sdk/inference/visualSegmenter.ts [hunk 14]
```diff
@@ -29,23 +29,22 @@ import { Model } from "clarifai-nodejs";
...
+                                    inputType: "Text"
                                 });
 
-
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath, 
-                                    inputType
+                                    filepath,
+                                    inputType: "Text",
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/visualSegmenter.ts]: **inputType** should be **image**

File: code_snippets/python-sdk/inference/diff_baseurl.py (hunk 1 above)
Comment (clarifai-m) [reply code_snippets/python-sdk/inference/diff_baseurl.py]: Reverted back to YOUR_PAT

**Comment by clarifai-m** [comment]
All comments has been resolved, please review again

File: code_snippets/python-sdk/inference/imageTextToText.ts [hunk 15]
```diff
@@ -42,14 +42,15 @@ const multiInputs = Input.getMultimodalInput({
...
+                                    inputType
                                 });
 
+
         Example for prediction through Filepath:
         const modelPrediction = await model.predictByFilepath({
-                                    filepath,
-                                    inputType: "Text",
+                                    filepath, 
+                                    inputType
```
Comment (srikanthbachala20) [inline code_snippets/python-sdk/inference/imageTextToText.ts]: Why are the inputType empty?

**Review by srikanthbachala20 — APPROVED** [review]
LGTM

**Pull request merged**